README.md

```diff
@@ -10,11 +10,12 @@ In most cases, Dockerfiles encode the instructions for ease-of-use: Just do `doc
 Currently supported packages:
 
 |                  | **Build instructions**                 | **Pre-built Docker image or binary files**                                                                                   |
-|------------------|----------------------------------------|------------------------------------------------------------------------------------------------------------------------------|
+| ---------------- | -------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------- |
 | **curl**         | [Github: oqs-demos/curl](curl)         | [Dockerhub: openquantumsafe/curl](https://hub.docker.com/repository/docker/openquantumsafe/curl)                             |
 | **Apache httpd** | [Github: oqs-demos/httpd](httpd)       | [Dockerhub: openquantumsafe/httpd](https://hub.docker.com/repository/docker/openquantumsafe/httpd)                           |
 | **nginx**        | [Github: oqs-demos/nginx](nginx)       | [Dockerhub: openquantumsafe/nginx](https://hub.docker.com/repository/docker/openquantumsafe/nginx)                           |
 | **Chromium**     | [Github: oqs-demos/chromium](chromium) | [Binary for Ubuntu 18.04](https://github.com/open-quantum-safe/oqs-demos/releases/download/v0.4.0/chromium-ubuntu-0.4.0.tgz) |
-| **HAproxy**        | [Github: oqs-demos/haproxy](haproxy)       | [Dockerhub: openquantumsafe/haproxy](https://hub.docker.com/repository/docker/openquantumsafe/haproxy)                           |
+| **HAproxy**      | [Github: oqs-demos/haproxy](haproxy)   | [Dockerhub: openquantumsafe/haproxy](https://hub.docker.com/repository/docker/openquantumsafe/haproxy)                       |
+| **OpenSSH**      | [Github: oqs-demos/openssh](openssh)   | *Dockerhub: openquantumsafe/openssh WIP*                                                                                     |
```

Comment:
This is because there's not yet a push step in the CCI config for this image. I will add this step referring to @baentsch above and remove the WIP tag and make it an actual link.